tests/install/install_test.go

```diff
@@ -19,28 +19,28 @@ package smoke_test
 import (
 	"os"
 
-	"gopkg.in/yaml.v2"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/rancher/elemental/tests/sut"
+	sut "github.com/rancher-sandbox/ele-testhelpers/vm"
 )
 
-type RancherOsConfig struct {
-	Rancheros Rancheros `yaml:"rancheros,omitempty"`
-}
+func checkOsAfterReboot(s *sut.SUT) {
+	// Reboot to check the installed OS
+	s.Reboot()
 
-type Rancheros struct {
-	Install Install `yaml:"install,omitempty"`
-}
+	By("Checking we booted from the installed OS")
+	ExpectWithOffset(1, s.BootFrom()).To(Equal(sut.Active))
```

Comment:
there is an `AssertBootedFrom` in the testhelpers that adds proper exit error if it doesnt match: https://github.com/rancher-sandbox/ele-testhelpers/blob/main/vm/sut.go#L555

Reply:
This piece of code was already here as-is, so I didn't touched it :D But I will give it a try.

Reply:
Done!